src/guide/diagrams-src/filters.puml

```diff
@@ -0,0 +1,26 @@
+@startuml
```

Comment:
About image, generated by this file. It should be called something like `data_filters.img`, not just `filters` because all this types works only with *data-events*.

Also if you want to show all *data-filters* here, then you should describe all other filters like: `PeerFilter`, `DomainFilter`, `AccountFilter` and etc.
It may be not so easy because of macro usage in our code. Be advised, that if you are using `VsCode` with `rust-analyzer` you can tap on `entity_filter!` macro, press `F1` and find `rust-analyzer: Expand macro recurcively` to see macro output that can be usefull for you